spec/controllers/finders_controller_spec.rb

```diff
@@ -90,5 +91,87 @@
         expect(response.status).to eq(404)
       end
     end
+
+    describe "viewing a policies finder with an a/b test" do
+      before do
+        content_store_has_item(
+          '/government/policies',
+            base_path: '/government/policies',
+            title: 'Policies',
+            details: {
+              facets: [],
+            },
+            links: {
+              organisations: [],
+            },
+        )
+
+        content_store_has_item(
+          '/government/policies/all',
+            base_path: '/government/policies/all',
+            title: 'All Policies',
+            details: {
+              facets: [],
+            },
+            links: {
+              organisations: [],
+            },
+        )
+
+        content_store_has_item(
+          '/government/policies/child-policy',
+            base_path: '/government/policies/child-policy',
+            title: 'Child Policy',
+            details: {
+              facets: [],
+            },
+            links: {
+              organisations: [],
+            },
+        )
+
+        rummager_response = %|{
+            "results": [],
+            "total": 0,
+            "start": 0,
+            "facets": {},
+            "suggested_queries": []
+          }|
+
+        stub_request(:get, "#{Plek.current.find('search')}/search.json?count=1000&fields=title,link,description,public_timestamp&order=-public_timestamp&start=0").to_return(status: 200, body: rummager_response, headers: {})
+      end
+
+      it "directs users in group A to the normal policies finder" do
+        setup_ab_variant("PolicyFinderTest", "A")
+
+        get :show, params: { slug: 'government/policies' }
+        finder_slug = subject.instance_variable_get(:@results).finder.slug
+
+        expect(response.status).to eq(200)
+        expect(finder_slug).to eq("/government/policies")
+      end
+
+      it "directs users in group B to the all policies finder" do
+        setup_ab_variant("PolicyFinderTest", "B")
+
+        get :show, params: { slug: 'government/policies' }
+        finder_slug = subject.instance_variable_get(:@results).finder.slug
+
+        expect(response.status).to eq(200)
+        expect(finder_slug).to eq("/government/policies/all")
+      end
+
+      it "directs users in both groups to a child policy finder" do
```

Comment:
This might be better worded to describe the case when a user goes directly to a finder that is not part of the a/b test. We had initially thought that you were trying to do an a/b/c test (which if you'd like to write the fastly VCL to do that please do!! :)  )